src/alert_rules/prometheus/prometheus_alerts.yaml

```diff
@@ -1,5 +1,5 @@
 "groups":
-- "name": "opensearch_dashboards.alerts"
+- "name": "wazuh_dashboard.alerts"
   "rules":
   - "alert": "OpenSearchDashboardsNotHealthy"
```

Comment:
Should it not be `WazuhDashboardsNotHealthy` instead of `OpenSearchDashboardsNotHealthy`

Reply:
I though about changing this but that would mean that, for consistency, I'd have to make a lot of additional changes making reintegrating changes from upstream more difficult, so I've decided to leave it as it is